extensions/cornerstone/src/getCustomizationModule.ts

```diff
@@ -3,8 +3,15 @@ import { toolNames } from './initCornerstoneTools';
 import DicomUpload from './components/DicomUpload/DicomUpload';
 import defaultWindowLevelPresets from './components/WindowLevelActionMenu/defaultWindowLevelPresets';
 import { colormaps } from './utils/colormaps';
+import { CONSTANTS } from '@cornerstonejs/core';
 
 const DefaultColormap = 'Grayscale';
+const { VIEWPORT_PRESETS } = CONSTANTS;
+const defaultVolumeQualityRange = {
```

Comment:
we don't need to name this object, can you just use those in the bottom and not put it in the top?